programs/survival/elytra_swap.sc

```diff
@@ -0,0 +1,17 @@
+// By right-clicking with an elytra or any chestplate this script will swap the items
+// By: IceWolf23
+
+global_swappable_items = {'netherite_chestplate','diamond_chestplate','iron_chestplate','golden_chestplate','chainmail_chestplate','leather_chestplate','elytra'};
+
+
+__on_player_uses_item(player, item_tuple, hand) -> (
+	if ( hand == 'mainhand' && (inventory_get(player, 38)) != null,
+		if ( global_swappable_items:(item_tuple:0) != null, (			
```

Comment:
Rogue parenthesis at end of line causes mismatch across whole script. Please delete.